docs/docs/configuration/license_plate_recognition.md

```diff
@@ -110,6 +123,70 @@ lpr:
       - "MN D3163"
 ```
 
+## Dedicated LPR Cameras
+
+Dedicated LPR cameras are single-purpose cameras with powerful optical zoom to capture license plates on distant vehicles, often with fine-tuned settings to capture plates at night.
+
+Users with a dedicated LPR camera can run Frigate's LPR by specifying a camera type of `lpr` in the camera configuration. An example config for a dedicated LPR camera might look like this:
+
+```yaml
+# LPR global configuration
+lpr:
+  enabled: True
+  min_area: 2000
+  match_distance: 1
```

Comment:
Does it make sense to specify match_distance if no known plates are specified?

Reply:
no, I'll change the example